integration/server_test.go

```diff
@@ -0,0 +1,36 @@
+// +build integration
+
+package integration
+
+import (
+	"os"
+	"os/exec"
+	"path"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	binaryName = "xds-relay"
+)
+
+func TestXdsClientGetsIncrementalResponsesFromUpstreamServer(t *testing.T) {
+	dir, err := os.Getwd()
```

Comment:
Is this guaranteed to be here in CI jobs? Should there be an environment variable but default to this if it is not set?

Reply:
yeah..the CI jobs are set up to correctly set the folder https://github.com/envoyproxy/xds-relay/blob/master/.github/workflows/integration-tests.yml#L40
They run locally too without any changes.